app/models/badges/supermentor_badge.rb

```diff
@@ -0,0 +1,9 @@
+class Badges::SupermentorBadge < Badge
+  seed 'Supermentor',
+    :legendary,
+    'supermentor',
+    'Mentored 100 students on a track'
```

Comment:
Do you want this to maybe also say something about the satisfaction rating?

Something like:

```suggestion
    'Satisfactorily mentored 100 students on a track'
```

Reply:
I think I'm ok with just "Mentored"